python-threatexchange/threatexchange/content_type/file_content.py

```diff
@@ -0,0 +1,30 @@
+import typing as t
+from content_base import ContentType
+from photo import PhotoContent
+from video import VideoContent
+
+class FileContent(ContentType):
+    """
+    Content representing a file. Determines if a file is a photo or video based on file extension.
+    """
+
+    VALID_PHOTO_EXTENSIONS = {".jpg", ".jpeg", ".png", ".gif"}
```

Comment:
For photos, we can align with the image types that Pillow supports. Check out this code:

```
from PIL import Image

exts = Image.registered_extensions()
```